src/features/call/hooks/useCallState.ts

```diff
@@ -0,0 +1,6 @@
+import { useAppSelector } from 'core/hooks';
+
+export default function useCallState(callId: number) {
```

Comment:
As you continue work on this, I think it would be good to add a return type to this function. Right now I believe the implicit (inferred) return type is `CallState | undefined | null`, and I find it a little bit weird and easy to make mistakes when it can return either null or undefined, and not just one of them (I would choose `null`).

Reply:
oof yep agreed, will put that into my refactor list of todos!